src/client/baseClient.ts

```diff
@@ -116,10 +128,58 @@ export class BaseClient {
       const domain = (baseMetrics.domain = await this.resolveDomain(
         this.config.uiCsdsServiceName
       ));
-      const resp = await this.performGetLambdasRequest(
-        lambdaRequestData,
+
+      const isV2 = this.isV2Domain(domain);
+
+      const resp = isV2
+        ? await this.performGetFunctionsRequest(
```

Comment:
I have separate functions performGetFunctionsRequest for V2 and old performGetLambdasRequest for V1 then once V1 is deprecate we just need to delete the performGetLambdasRequest,

Reply:
Did you also add a comment about this in code ? So we can remember moving forward. You can add it as // TODO:

Reply:
Just added a comment there to remember